mock_server/index.js

```diff
@@ -6,7 +6,7 @@ const wss = new WebSocketServer.Server({ port: 3210 })
 // Creating connection using websocket
 wss.on("connection", ws => {
     console.log("new client connected");
-    let rawData = fs.readFileSync('sample_data.json');
```

Comment:
should we revert this? or just update sample_data.json with the new data if its better instead of commiting a second json  file

Reply:
Addressed